src/main/java/taxi/dao/impl/CarDaoImpl.java

```diff
@@ -238,7 +238,7 @@ private Car parseCarFromResultSet(ResultSet resultSet) throws SQLException {
         String manufacturerCountry = resultSet.getNString("manufacturer_country");
         Manufacturer manufacturer = new Manufacturer(manufacturerName, manufacturerCountry);
         manufacturer.setId(manufacturerId);
-        long carId = resultSet.getLong("id");
+        long carId = resultSet.getObject("id", Long.class);;
```

Comment:
те ж саме на 236 строці
```suggestion
        Long carId = resultSet.getObject("id", Long.class);;
```